website/content/docs/core/controller/mcp.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Agentica > Guide Documents > Core Library > Function Controllers > MCP Protocol
+---
+
+
+## MCP SDK
+
+```typescript filename="src/main.ts" showLineNumbers copy
```

Comment:
I added it
but this interface follow next https://github.com/wrtnlabs/agentica/pull/360